packages/shared/src/lib/urls.ts

```
@@ -5,6 +5,24 @@ import {NavItem} from '@shared/types/urls';
 
+export function isValidUrl(url: string): boolean {
+  try {
+    new URL(url);
+    return true;
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  } catch (error1) {
+    // Passthrough.
+  }
+
+  try {
+    new URL('https://' + url);
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)